finalPrep/annotations.js

```diff
@@ -2,41 +2,61 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+//this is a function with 5 parameters (name, age, fur, clothes, and specialPower)
 function buildABear(name, age, fur, clothes, specialPower) {
-  var greeting = `Hey partner! My name is ${name} - will you be my friend?!`
-  var demographics = [name, age]
-  var powerSaying = "Did you know that I can " + specialPower + " ?"
+  //declare a variable of greeting- value is a string with the name parameter interpolated in
+  var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+  //declare a variable of demographics - value is an array with two elements which are the parameters name and age
+  var demographics = [name, age];
+  //declare a variable of powerSaying - value is a string with the specialPower parameter interpolated in
+  var powerSaying = "Did you know that I can " + specialPower + " ?";
+  //this is an object with the name builtBear that holds the attributes basicInfo, clothes, exterior, cost, sayings and isCuddly
   var builtBear = {
     basicInfo: demographics,
     clothes: clothes,
     exterior: fur,
```

Comment:
You could take note of where different keys get their values- are they passed in as params, or declared as variables above?